app/src/main/res/values-de/strings.xml

```diff
@@ -1231,38 +1232,38 @@ registriert. Bitte versuchen Sie es mit einer anderen.</string>
     <string name="record_audio_max_file_size_reached_title">Aufnahme wurde beendet</string>
     <string name="record_audio_max_file_size_reached_text">Die Dateigröße für Audionachrichten ist auf %1$d MB begrenzt.</string>
     <string name="record_audio_unable_due_to_ongoing_call">Sie können während eines Anrufs keine Audionachricht aufnehmen.</string>
-    <string name="permission_dialog_title">App-Berechtigungen</string>
-    <string name="call_permission_dialog_description">Um einen Anruf zu tätigen, erlauben Sie Wire den Zugriff auf Ihr Mikrofon und die Suche nach Geräten in der Nähe in Ihren Geräteeinstellungen.</string>
+    <string name="permission_dialog_title">App permissions</string>
+    <string name="call_permission_dialog_description">To make a call, allow Wire to access your microphone and to find nearby devices in your device settings.</string>
```

Comment:
This is changed to english

Reply:
@ohassine Please sync with Astrid or in crowdin to check for the missing translation, then we can retrigger the action =)